components/mpas-ocean/src/shared/mpas_ocn_surface_bulk_forcing.F

```diff
@@ -880,6 +886,215 @@ subroutine ocn_surface_bulk_forcing_active_tracers_subglacial_runoff(meshPool, f
+      call mpas_pool_get_array(forcingPool, 'snowFlux', snowFlux)
+      call mpas_pool_get_array(forcingPool, 'riverRunoffFlux', riverRunoffFlux)
+      call mpas_pool_get_array(forcingPool, 'iceRunoffFlux', iceRunoffFlux)
+      call mpas_pool_get_array(forcingPool, 'seaIceFreshWaterFlux', seaIceFreshWaterFlux)
+      call mpas_pool_get_array(forcingPool, 'icebergFreshWaterFlux', icebergFreshWaterFlux)
+      call mpas_pool_get_array(forcingPool, 'landIceFreshwaterFlux', landIceFreshwaterFlux)
+
+      nCells = nCellsArray( 3 )
+      if (associated(indexRain) .and. associated(rainFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFlux(indexRain, iCell) = rainFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexRain is not associated')
+      endif
+      if (associated(indexSnow) .and. associated(snowFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFlux(indexSnow, iCell) = snowFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexSnow is not associated')
+      endif
+      if (associated(indexIceRunoff) .and. associated(iceRunoffFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFluxRunoff(indexIceRunoff, iCell) = iceRunoffFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexIceRunoff is not associated')
+      endif
+      if (associated(indexRiverRunoff) .and. associated(riverRunoffFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFluxRunoff(indexRiverRunoff, iCell) = riverRunoffFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        if (.not. associated(indexRiverRunoff)) call mpas_log_write('indexRiverRunoff is not associated')
+        if (.not. associated(riverRunoffFlux)) call mpas_log_write('riverRunoffFlux is not associated')
+      endif
+      if (associated(indexSeaIceFreshWater) .and. associated(seaIceFreshWaterFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFlux(indexSeaIceFreshWater, iCell) = seaIceFreshWaterFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexSeaIceFreshWater is not associated')
+      endif
+      if (associated(indexIcebergFreshWater) .and. associated(icebergFreshWaterFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFlux(indexIcebergFreshWater, iCell) = icebergFreshWaterFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexIcebergFreshWater is not associated')
+      endif
+      if (associated(indexLandIceFreshwater) .and. associated(landIceFreshwaterFlux)) then
+        do iCell=1,nCells
+          freshwaterSurfaceFlux(indexLandIceFreshwater, iCell) = landIceFreshwaterFlux(iCell) / rho_sw
+        enddo  !  iCell
+      else
+        call mpas_log_write('indexLandIceFreshwater or landIceFreshwaterFlux is not associated')
+      endif
+      call mpas_log_write('end fw assign sflux')
```

Comment:
I think this should probably also be removed.

Reply:
Thanks for catching that!